src/tools/python/variables.md

```diff
@@ -1,23 +1,20 @@
 # Variables
 
-We've already gotten a taste of variables from previous lessons inside datatypes, because its difficult to have good examples without them.
+We have already seen in the previous sections that we can print out *values* and *expressions* of values that are evaluated by Python. However, it is often necceasry to remember a given value beyond the scope of a singular print statement or line of code. This is what motivates the need for a __variable__, which stores elements in computer memory.
 
+You can think of a variable as a "storage box" that has some kind of "datatype" assigned to them. Each of these boxes has a value associated with it. 
 
-You can think of a variable as a "storage box" that has some kind of "type" assigned to them. Each of these boxes has a value associated with it. 
-
- ##### *see the previous datatype section about the different types a variable can be*
-
-Here is how you assign a variable, or fill that "box"
+Here is how you assign a variable, or fill that "box." This is known as an assignment statement, or an initialization statement if this is the first time you are introducing a variable. 
 ```py,norepl
 a = 1
 ```
 
 lets break it down character by charater:
 - `a`: the variable name
 - `=`: this keyword is how we assign to variables. Most notably, is that it is explicitly assignment. You will learn about `==` in the next section, conditionals, but it always **sets** the left side **equal to** the right side, never the other way
-- `1` the value of which we are assigning the variable `a` to
+- `1` the value of which we are assigning the variable `a` to. For now, we will assgin a to be an integer, or type `int` (a nonfractional number). We will go over the different datatypes that can be taken on in the next section.
```

Comment:
![image](https://github.com/user-attachments/assets/fdb561bb-d354-4f50-b2e7-b6cb8676c55a)
currently in your PR from `SUMMARY.md` variables comes before datatypes